### ISSUE TITLE
Foundation: Void tag description

### ISSUE THREAD

**Issue by fabulousgk** [TheOdinProject/curriculum#26839]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Currently, the lesson has the following paragraph:
> There are some HTML elements that do not have a closing tag. These elements often look like this: <br /> or <img/>, but some can also be used without the closing forward slash such as <br> or <img>. These are known as self-closing tags or empty elements because they don’t wrap any content. We will encounter a few of these in later lessons, but for the most part, elements will have both opening and closing tags.

I think we need to include the definition for void tags and clarify that self-closing tags, while still prevalent are not actually part of the HTML 5 spec and are discouraged by the spec.

> Some HTML elements do not have a closing tag. These elements use a single tag, called a void tag. You may still find such tags formatted as self-closing tags, with a closing / (<br />, <img />), which are a holdover from a previous HTML specification.  However, the current HTML 5 spec discourages the use of self-closing tags, and you should use the void-tag syntax instead. We will encounter a few of these in later lessons, but for the most part, elements will have both opening and closing tags.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-elements-and-tags

### (Optional) Discord Name

therealpibbles

**Comment by fabulousgk** [comment]
Yeah, that makes sense.  If you will assign it, I will work on it for a PR.

**Comment by KevinMulhern** [comment]
Thanks @fabulousgk, its all yours 🚀
